config/workflows/geo_workflow.json

```diff
@@ -0,0 +1,127 @@
+{
+  "workflows": [
+    {
+      "name": "geo_works",
```

Comment:
We should create a ticket to decide whether we need different book and geo workflows.  Having two workflows that differ only by their name/labels is a good placeholder in case we decide to change one later.  But if we really do want to use the same workflow, it would be nice to remove the duplication here.